react-components/src/context/NavigationContext.js

```diff
@@ -72,6 +75,10 @@ const NavigationContextProvider = props => {
 
     const showAccountCreated = () => NavigationActions.showAccountCreated({ dispatch, t });
 
+    const showAddressBook = () => {
+        window.location.href = navigationState.addressBookPath;
```

Comment:
Shouldn't we rather use `history.push` here? See https://reactrouter.com/web/api/Hooks/usehistory.

Reply:
currently, the `react-router-dom` is only installed in venia repo not in core components repo, are you happy with how the router is used in venia repo? I tested in my local, it seems that browser controls history works as well, but I will have a look for sure, thank you @herzog31 :)

Reply:
We can install `react-router-dom` in `aem-core-cif-components`, but we have to put it as `peerDependency`. But as long as the feature works without it...

Reply:
Browser history should work with `window.location.href`, just wondering if it has any side effects in terms of navigating within React. Probably only works for navigating between actual AEM pages.

Reply:
Hi @herzog31, I tried to use `useHistory` and then `history.push`, but since we don't have React route context introduced in core components repo yet, it doesn't seem working at the moment, so are you happy for leaving this logic as is at the moment or do you have other thoughts on this? :)

Reply:
If React navigation works with it, let's keep it :)

Reply:
Thanks for that @herzog31 :)